molecule/default/tests/test_reboot_devices.py

```diff
@@ -0,0 +1,58 @@
+import os
+import re
+import pytest
+import testinfra.utils.ansible_runner
+
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('dashboard_hosts')
+
+pre_cmd = "sudo bash -c \"source /root/openrc; "
+
+
+@pytest.mark.jira('asc-228')
+@pytest.mark.skip(reason='add skipped tests for asc-301')
+def test_to_reboot_nova_compute(host):
+    cmd = pre_cmd + "ansible -m shell nova_compute -a \'reboot\' \""
+    assert host.run_expect([0], cmd)
+
+
+@pytest.mark.jira('asc-228')
+@pytest.mark.skip(reason='add skipped tests for asc-301')
+def test_to_reboot_log_hosts(host):
+    cmd = pre_cmd + "ansible -m shell log_hots -a \'reboot\' \""
+    assert host.run_expect([0], cmd)
+    verify_the_device_is_backup(host)
```

Comment:
I don't see this method defined, is it an existing testinfra fixture?

Reply:
Will add it in the next commit